python/ray/tune/experimental/output.py

```diff
@@ -512,6 +521,10 @@ def _get_dict_as_table_data(
             if include and key not in include and k not in include:
                 continue
 
+            # `include` is a dict (mapping actual keys to to-be-displayed-keys).
+            if isinstance(include, dict):
+                k = include[k]
```

Comment:
Can this mapping be done with a separate `display_key_mapping: Dict[str, str]` param instead of using the implicit `include`, which is a `Collection`?

Reply:
`include` is always(!) the `self._progress_metrics` property (or None). Maybe we should clean this up a little to make this more unified and simpler ...

Reply:
Fixed.